src/components/ChatRoom.js

```diff
@@ -57,14 +57,33 @@ export default function ChatRoom() {
       });
   }, []);
 
+  const handleDelete = (id) => {
+    const token = localStorageService.get("auth-token");
+    let config = {
+      method: "delete",
+      maxBodyLength: Infinity,
+      url: BaseUrl + "chat/chatroom/" + id + "/",
+      headers: {
+        Authorization: `token ${token}`,
+      },
+    };
+    axios.request(config)
+    .then(() => {
+ 
```

Comment:
Here once the chatroom is deleted, can you please show a confirmation message to the user that the chatroom has been deleted.